src/content/docs/en/guides/troubleshooting.mdx

```diff
@@ -199,6 +199,29 @@ const answer = sum(2, 4);
 <Debug {answer} />
 ```
 
+## Creating minimal reproductions
+
+When you encounter a bug in Astro, it's most helpful to create a minimal reproduction of the issue. This helps us to understand the context, quickly identify the problem and fix it.
+
+A minimal reproduction should be a simplified version of the code that reliably demonstrates the issue.
+
+### Create a StackBlitz via [astro.new](https://astro.new)
+
+You can use [astro.new](https://astro.new) to create a new Astro project with a single click. There you can choose a couple of options to setup a new project, for minimal reproductions, we recommend the minimal (empty) example, so you can isolate the occuring issue.
+
+We ask you to use [StackBlitz](https://stackblitz.com) because it allows maintainers and people willing to help quickly reproduce your issue without setting up a local environment. Whereas with a complete project or repo, we would have to clone it, install dependencies, and run it locally, which can introduce more issues.
```

Comment:
To be honest... yes. It requires more setup to ensure it _is_ a good repro (i.e. is small enough, isn't just an Astro versioning issue). It's also helpful as a handoff so we can modify the code in-place. If we find a fix while playing, we can send a modified link back to the issuer to say "this change seemed to resolve the issue. Let me know if it works on your end."

Reply:
Same opinion as Ben. The easier it is to get the reproduction started, the easiest it is. Stackblitz especially works well for this because it starts really quickly, but other options like CodeSandbox are also better than repos in most cases